src/lib/core/CHIPTLV.h

```diff
@@ -58,6 +58,16 @@ class PacketBuffer;
 namespace chip {
 namespace TLV {
 
+inline uint8_t operator|(TLVElementType lhs, TLVTagControl rhs)
```

Comment:
Style nit: but strongly prefer white space around `operator` the operation `|` to improve readability.

Reply:
can we make .clang-format do that?  if not, we're gonna have trouble enforcing

Reply:
Agreed. I can't seem to find anything in the clang-format documentation that indicates control over this specific behavior.

Reply:
As in, whitespace between `|` and `(`?  Our clang-format configuration will go ahead and remove it if I add it in...  Maybe we could change that configuration?